src/components/skip/skip.js

```diff
@@ -36,9 +36,11 @@ class Skip extends Component {
   render(): React$Element<any> | void {
     if (this.props.playerSize !== PLAYER_SIZE.TINY && !this.props.loading) {
       return (
-        <div tabIndex="0" aria-label={this.props.label} className={skipStyle.btnSkip} onClick={this.props.onClick}>
-          <Text id={this.props.label} />
-        </div>
+        <Localizer>
```

Comment:
Localizer is part of the fix? how?

Reply:
https://github.com/synacor/preact-i18n#localizer

Reply:
I know what is Localizer. just asked if it's relevant to the aria label only or it should be added also before, for the label itself?

Reply:
relevant to the aria label only
<Localizer /> is a Compositional Component. It "renders" out any <Text /> values **in its child's props.**
a quote from the link